Crashes on node v12

In the below code we check for `error.message` to end with the `customPath`.

https://github.com/dougmoscrop/serverless-plugin-split-stacks/blob/53326465b7955f9d0e3ebeef1f0d310f5197cab5/lib/migration-strategy/custom.js#L16

But starting with node.js v12, for `MODULE_NOT_FOUND` errors the error message also includes the require stack (https://github.com/nodejs/node/pull/25690), which breaks the expected behavior that we want. 

```js
// e.message before node v12
"Cannot find module '/<serviceDirectory>/stacks-map.js'"
```

```js
// e.message in node v12
"Cannot find module '/<serviceDirectory>/stacks-map.js'
Require stack:
- /<serviceDirectory>/node_modules/serverless-plugin-split-stacks/lib/migration-strategy/custom.js
- /<serviceDirectory>/node_modules/serverless-plugin-split-stacks/lib/migrate-new-resources.js
- /<serviceDirectory>/node_modules/serverless-plugin-split-stacks/split-stacks.js
- /<serviceDirectory>/node_modules/serverless/lib/classes/PluginManager.js
- /<serviceDirectory>/node_modules/serverless/lib/Serverless.js
- /<serviceDirectory>/node_modules/serverless/lib/utils/autocomplete.js
- /<serviceDirectory>/node_modules/serverless/bin/serverless"
```

--- a/lib/migration-strategy/custom.js
+++ b/lib/migration-strategy/custom.js
@@ -13,7 +13,7 @@ function loadCustomizations(config) {
     return require(customPath);
   } catch (e) {
     // If module not found ignore, otherwise crash
-    if (e.code !== 'MODULE_NOT_FOUND' || !e.message.endsWith(`'${customPath}'`)) {
+    if (e.code !== 'MODULE_NOT_FOUND' || !(e.message.split('\n')[0].endsWith(`'${customPath}'`))) {
       throw e;
     }
 

